.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "libraries/fc"]
 	path = libraries/fc
 	url = https://github.com/steemit/fc.git
+[submodule "libraries/plugins/rocksdb/rocksdb"]
+	path = libraries/plugins/rocksdb/rocksdb
+	url = https://github.com/vogel76/rocksdb.git
```

Comment:
even if we were going to use a submodule (we're not using submodules or anything of the sort), pointing at a private fork in a personal account is a hard no.

Reply:
The fork has been created because I had to fix one bug in rocksdb/CMakeLists.txt which disallow using this project as part of other project. And this is not private repo, but public one. Made changes were accepted and merged to the rocksdb master branch, but still didn't appear in any release.
We can of course make dedicated steem repo for it (even I don't see good reason for it because it is rather temporary state - until change will be published in regular rocksdb release), but we still need a way to integrate it to the main steem project (so use rejected by you git subtree or git subrepo).

Reply:
I think you misunderstood me.

No submodules.

No subtrees.

**Please vendor it in.  ASAP. **

(This also solves the other problem you describe (needing a local edit that's not in an upstream release.))